packages/react-scripts/scripts/start.js

```diff
@@ -42,6 +42,11 @@ if (!checkRequiredFiles([paths.appHtml, paths.appIndexJs])) {
   process.exit(1);
 }
 
+process.stdin.on('end', function() {
```

Comment:
Do you think it's worth toggling this with an env var? Webpack uses `--stdin` to turn this behaviour on - although testing worked fine, I don't want to inadvertently break some other scenarios.